openshift/templates/rproxy-environment-template.json

```diff
@@ -178,41 +276,149 @@
       "name": "APP_DEPLOYMENT_TAG",
       "displayName": "Image tag to watch for changes and trigger deployment.",
       "description": "Image tag to watch for changes and trigger deployment.",
-      "value": "prod",
+      "value": "latest",
       "required": true
     },
     {
       "name": "NGINX_PROXY_HOST",
-      "displayName": "NGinx Proxy Host",
+      "displayName": "Nginx Proxy Host",
       "description": "The HTTP Header Host value we want to give to the downstream HTTP server",
-      "value": "projects.eao.gov.bc.ca",
+      "value": "esm-master.pathfinder.gov.bc.ca",
       "required": true
     },
     {
       "name": "NGINX_PROXY_URL",
       "displayName": "Host that receives incoming web requests",
-      "description": "The HTTP Header Host value we want to give to the downstream HTTP server",
-      "value:": "http://esm-prod:3000",
+      "description": "The default URL to use when proxying requests to the application's API.",
+      "value:": "http://epic-master:3000",
       "required": true
     },
     {
       "name": "HTTP_API_DEPLOY_CONFIG",
       "displayName": "A way to inject nginx params into the config",
-      "description": "The HTTP Header Host value we want to give to the downstream HTTP server"
+      "description": ""
     },
     {
       "name": "NGINX_EXPORT_URL",
-      "displayName": "Geo Service location",
-      "description": "The HTTP Header Host value we want to give to the downstream HTTP server",
+      "displayName": "Geo Export Service location",
+      "description": "The default URL to use when proxying requests to the GEO-DATA-EXPORT microservice.",
       "value": "http://eao-geo-service:3000/",
       "required": true
     },
+    {
+      "name": "HTTP_API_EXPORT_DEPLOY_CONFIG",
+      "displayName": "A way to inject Nginx params into the config",
+      "description": ""
+    },
+    {
+      "name": "NGINX_GEOCODER_URL",
+      "displayName": "Geocoder Service location",
+      "description": "The default URL to use when proxying requests to the GEOCODER microservice.",
+      "required": true
+    },
+    {
+      "name": "NGINX_GEOCODER_API_KEY",
+      "displayName": "The API Key to authenticate against the GEOCODER microservice",
+      "description": ""
+    },
+    {
+      "name": "HTTP_API_GEOCODER_DEPLOY_CONFIG",
+      "displayName": "A way to inject nginx params into the config",
+      "description": ""
+    },
     {
       "name": "NGINX_PROXY_URL_PUBLIC",
       "displayName": "Host that receives incoming www.* requests",
+      "description": "The default URL to use when proxying requests to the PUBLIC microservice.",
+      "value": "http://eao-public:8080",
+      "required": true
+    },
+    {
+      "name": "NGINX_PROXY_PASS_PUBLIC",
```

Comment:
I see *NGINX_PROXY_PASS_PUBLIC* and *NGINX_PROXY_URL_PUBLIC* have the same value and are being used to proxy the traffic between the same destination and source route, although on the public and non-public server definitions in the config file.
Just wondering if this is by design, and why, or we can safely merge them into a single parameter: I realize this may be something we "inherited", if we can do some clean up it would be great.

Reply:
You're right. This is something we inherited, so I didn't want to mess with it. I'm happy to change/remove the extra configuration variables. Let's discuss offline.